pandas/core/sorting.py

```diff
@@ -208,7 +208,7 @@ def lexsort_indexer(keys, orders=None, na_position="last"):
             cat = Categorical(key, ordered=True)
 
         if na_position not in ["last", "first"]:
-            raise ValueError("invalid na_position: {!r}".format(na_position))
+            raise ValueError(f"invalid na_position: {na_position}")
```

Comment:
```suggestion
            raise ValueError(f"invalid na_position: {repr(na_position)}")
```

Reply:
@WillAyd these seem excessive, adding repr on a boolean is not worth IMHO

Reply:
Ah makes sense. Oversight on my end